src/elements/a-gltf-entity.js

```diff
@@ -121,7 +121,18 @@ AFRAME.registerElement("a-gltf-entity", {
         // If the src attribute is a selector, get the url from the asset item.
         if (src.charAt(0) === "#") {
           const assetEl = document.getElementById(src.substring(1));
-          src = assetEl.getAttribute("src");
+
+          const fallbackSrc = assetEl.getAttribute("src");
+          const highSrc = assetEl.getAttribute("high-src");
+          const lowSrc = assetEl.getAttribute("low-src");
+
+          if (highSrc && AFRAME.quality === "high") {
```

Comment:
Are we planning to submit `a-gltf-entity` and `a-progressive-asset` to A-Frame core? It feels a bit presumptive to claim `AFRAME.quality` if we aren't. I'd prefer we use our own global instead, like `APP.quality` or even `SOCIAL.quality`.

Reply:
In the component inflation branch we introduce the `AFRAME.AGLTFEntity` namespace, seems like we should put this in there so that we can release `a-progressive-asset` and `a-gltf-entity` as its own  "enchained GLTF loading" package.